README.md

```diff
@@ -225,6 +225,13 @@ environment variables.
    * Alternative URL for the collector, this is mostly used for debugging and
      experimentation with newer collectors.
 
+Alternatively a configuration may be specified in the root of the JAR with a
+standard properties format (`key=value`) which is named `iopipe.properties`.
+The configuration values are only used if they have not been specified by
+system properties or environment variables. Generally using this is not
```

Comment:
If this is not recommended, why offer the instructions?

Reply:
I have it there to just say that it is available for use, otherwise nobody would know about it.